cmd/private-org-peribolos-sync/main_test.go

```diff
@@ -0,0 +1,277 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"k8s.io/test-infra/prow/config/org"
+	"k8s.io/test-infra/prow/github"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/config"
+)
+
+type fakeGitHub interface {
+	GetRepo(owner, name string) (github.FullRepo, error)
+}
+
+type fakeGitHubClient struct {
+	fakeGitHub
+}
+
+func (f fakeGitHubClient) GetRepo(owner, name string) (github.FullRepo, error) {
```

Comment:
Does the upstream `prow/github` not have this fake already?

Reply:
surprisingly no!

Reply:
I can add this in upstream and use it here in a follow-up.

Reply:
opened https://github.com/kubernetes/test-infra/pull/17164

Reply:
addressed. The code is using the upstream fakegithubclient.